CreateService/handler.ts

```diff
@@ -180,18 +216,25 @@ export function CreateServiceHandler(
       productName
     )
       .chain(subscription =>
-        createServiceTask(
-          getLogger(context, logPrefix, "CreateService"),
+        getUserTask(
+          getLogger(context, logPrefix, "GetUser"),
           apiClient,
-          servicePayload,
-          subscriptionId,
-          (sandboxFiscalCode as unknown) as FiscalCode
-        ).map(service =>
-          ResponseSuccessJson({
-            ...service,
-            primary_key: subscription.primary_key,
-            secondary_key: subscription.secondary_key
-          })
+          userAttributes.email
+        ).chain(userInfo =>
+          createServiceTask(
+            getLogger(context, logPrefix, "CreateService"),
+            apiClient,
+            servicePayload,
+            subscriptionId,
+            (sandboxFiscalCode as unknown) as FiscalCode,
```

Comment:
why do we need this cast here ? ) it looks like it's already a FiscalCode

Reply:
`sandboxFiscalCode` is a `NonEmptyString` ( 'cause we are reading it from `.env` ). Can we change its type in other ways?